### PR TITLE
fix: notification icon default styling

### PR DESCRIPTION
**Pull request by rmathis**

### CODE REVIEW

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 1]
```diff
@@ -144,20 +142,83 @@ export default {
...
+      border: 1px solid var(--notif-icon-fg-color, #FFFFFF);
+      border-radius: 50%;
+      display: block;
+      width: 1.5rem;
+      height: 1.5rem;
+      font-size: 1.1rem;
+      line-height: 1.2rem;
+      position: absolute;
+      top: -.25rem;
+      right: -.5rem;
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]: :art: 
```suggestion
      right: -0.5rem;
```

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 2]
```diff
@@ -144,20 +142,83 @@ export default {
...
+    .count {
+      border: 1px solid var(--notif-icon-fg-color, #FFFFFF);
+      border-radius: 50%;
+      display: block;
+      width: 1.5rem;
+      height: 1.5rem;
+      font-size: 1.1rem;
+      line-height: 1.2rem;
+      position: absolute;
+      top: -.25rem;
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]: :art: 
```suggestion
      top: -0.25rem;
```

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue (hunk 1 above)
Comment (rmathis) [reply notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]: done

**Comment by drewwills** [comment]
icon1

Just excellent -- this is a huge improvement.

icon2

What if we `display: none;` on the badge if the count is zero?  Is that easy?  I'm concerned that the number+circle isn't very clear when superimposed on the bell, white-on-white.

**Comment by drewwills** [comment]
Thanks @rmathis

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="notification-icon">
-    <dropdown id="_uid" no-caret :variant="variant">
+    <dropdown id="_uid" no-caret toggle-class="btn-icon">
       <template slot="button-content">
-        <font-awesome-icon icon="bell"/>
-        <span class="count">{{ count }}</span>
+        <font-awesome-icon icon="bell" size="2x" fixed-width/>
+        <span class="count" :class="{ 'alert': count > 1, 'hidden': count <=0 }">{{ countDisplay }}</span>
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]: :question: `'alert': count > 1` two notifications is when this becomes significant? Should this be `'alert': count > 0`?
:art: : `'hidden': count <=0` I have a mild stylist preference for using `<` over `<=`,  `'hidden': count < 1`

**Pull request merged**